cmd/server/handler_client_factory.go

```diff
@@ -40,6 +40,12 @@ func newClientManager(c *config.Config) client.Manager {
 		}
 		m.Watch(context.Background())
 		return m
+	case *config.RedisConnection:
+		m := &client.RedisManager{
```

Comment:
No, I've kept everything quite similar to the memory manager so it's all done in Hydra.